cms/static/sass/partials/cms/theme/_extras.scss

```diff
@@ -0,0 +1,4 @@
+// Extra rules needed for a Studio theme
+//
+// For your own theme, you can add rules directly into an _extra.scss
+// file. You can also import additional partials into it.
```

Comment:
Perhaps add a little more description about how this is the override file that loads last and therefore you should be able to use any variables, functions and mixins elsewhere in the sass.

Reply:
_last.scss ? (feel free to ignore)